src/Http/Controllers/AssetsController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Response as FacadesResponse;
+
+class AssetsController extends Controller
+{
+    private const ASSETS_PATH = "static";
```

Comment:
Maybe it should be a StaticCotroller? Assets and static words should be unified.